src/validation.cpp

```diff
@@ -2130,13 +2134,12 @@ bool CChainState::ConnectBlock(const CBlock& block, BlockValidationState& state,
             return state.Invalid(BlockValidationResult::BLOCK_CONSENSUS, "bad-blk-sigops");
         }
 
-        txdata.emplace_back(tx);
```

Comment:
small bonus is that precompute data isn't computed unless `fScriptChecks` is true

Reply:
Oh, that's great. I never realized that.

Reply:
That is a nice bonus. That's millions of hashes saved over the course of an IBD. Not sure how meaningful that is.

Reply:
freebie is free